pleiades/vocabularies/setuphandlers.py

```diff
@@ -113,6 +113,31 @@ def installVocabularies(context):
             settings.arch_remains.append(dict(id=vocab_entry['id'],
                                               title=vocab_entry['title']))
 
+    # prepopulate relationship_types vocab - if uninitialized
+    if not settings.relationship_types:
+        settings.relationship_types = []
+        vocab_data = [
+            {'id': 'connection', 'title': 'connection', 
+             'same_as' : '', 'hidden' : False},
+            {'id': 'at', 'title': 'at', 
+             'same_as' : '', 'hidden' : False},
+            {'id': 'on', 'title': 'on', 
+             'same_as' : '', 'hidden' : False},
+            {'id': 'part_of_admin', 'title': 'part of (administrative)', 
+             'same_as' : '', 'hidden' : False},
+            {'id': 'part_of_regional', 'title': 'part of (regional)', 
+             'same_as' : '', 'hidden' : False},
+            {'id': 'near', 'title': 'near', 
+             'same_as' : '', 'hidden' : False},
+            {'id': 'intersects', 'title': 'intersects', 
+             'same_as' : '', 'hidden' : False},
+        ]
+        for vocab_entry in vocab_data:
+            settings.relationship_types.append(dict(id=vocab_entry['id'],
+                                              title=vocab_entry['title'],
```

Comment:
actually I got a bunch of issues when trying to do it differently:
tracebacks such as:
```
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module plone.z3cform.layout, line 66, in __call__
  Module plone.z3cform.layout, line 50, in update
  Module plone.z3cform.fieldsets.extensible, line 59, in update
  Module plone.z3cform.patch, line 30, in GroupForm_update
  Module z3c.form.group, line 132, in update
  Module z3c.form.form, line 135, in updateWidgets
  Module z3c.form.field, line 277, in update
  Module z3c.form.browser.multi, line 63, in update
  Module z3c.form.browser.widget, line 171, in update
  Module z3c.form.widget, line 479, in update
  Module z3c.form.widget, line 131, in update
  Module z3c.form.widget, line 474, in value
  Module collective.z3cform.datagridfield.datagridfield, line 147, in updateWidgets
  Module z3c.form.widget, line 416, in updateWidgets
  Module z3c.form.widget, line 393, in applyValue
  Module collective.z3cform.datagridfield.datagridfield, line 257, in set
  Module z3c.form.object, line 208, in updateWidgets
  Module collective.z3cform.datagridfield.autoform, line 60, in update
  Module z3c.form.object, line 87, in update
  Module plone.z3cform.patch, line 21, in BaseForm_update
  Module z3c.form.form, line 153, in update
  Module collective.z3cform.datagridfield.datagridfield, line 315, in updateWidgets
  Module collective.z3cform.datagridfield.autoform, line 40, in updateWidgets
  Module z3c.form.form, line 135, in updateWidgets
  Module z3c.form.field, line 243, in update
  Module z3c.form.datamanager, line 102, in canWrite
  Module z3c.form.datamanager, line 66, in adapted_context
TypeError: ('Could not adapt', [{'id': 'unknown', 'title': 'Unknown'}, {'id': 'none', 'title': 'None'}, {'id': 'traces', 'title': 'Traces'}, {'id': 'substantive', 'title': 'Substantive'}, {'id': 'restored', 'title': 'Restored'}, {'id': 'notvisible', 'title': 'Not visible'}], <InterfaceClass pleiades.vocabularies.interfaces.IVocabTerm>)
```